boms/cloud-oss-bom/RELEASING.md

```diff
@@ -95,13 +95,10 @@ new release is available on Maven Central.
     * `$ repo --title .*v16.4.0 tag automerge`
 * Manually edit and update any pom.xml files in https://github.com/GoogleCloudPlatform/java-docs-samples that dependabot missed
 * In google3 run:
-    * `$ g4d -f bom`
-    * `$ OLD=19.0.0; NEW=19.1.0`
-      (Adjust the old and new versions)
-    * `/google/src/head/depot/google3/devtools/scripts/replace_string "&lt;version>${OLD}&lt;/version>" "&lt;version>${NEW}&lt;/version>"`
-    * `/google/src/head/depot/google3/devtools/scripts/replace_string "&lt;version&gt;${OLD}&lt;/version&gt;" "&lt;version>${NEW}&lt;/version>"`
-    * `/google/src/head/depot/google3/devtools/scripts/replace_string "&lt;version&gt;${OLD}&lt;/version>" "&lt;version>${NEW}&lt;/version>"`
-    * Sanity check the cl and send it for review.
+    * `$ scripts/update_docs.sh <old version> <new version>`
+      * For example, `$ scripts/update_docs.sh 16.3.0 16.4.0`
+      * When asked whether to add changes to the first CL, answer yes.
```

Comment:
Do we need this step? I don't think I've been asked this?

Reply:
Strange. I was asked by the 2nd invocation of `/google/src/head/depot/google3/devtools/scripts/replace_string`.

Reply:
Maybe there were no match for the 2nd and 3rd replace_string command when you ran it.